pkgs/data/icons/zafiro-icons/default.nix

```diff
@@ -26,10 +27,26 @@ stdenv.mkDerivation rec {
 
   dontDropIconThemeCache = true;
 
+  dontPatchELF = true;
+  dontRewriteSymlinks = true;
+
   installPhase = ''
+    runHook preInstall
+
+    # remove spaces from filesnames because that causes gtk-update-icon-cache to
+    # fail
+    while read file; do
+      mv "$file" "$(tr ' ' - <<<$file)"
+    done < <( find . -name "* *" )
```

Comment:
```suggestion
    # remove copy file, as it is there clearly by mistake
    rm "apps/scalable/android-sdk (copia 1).svg"
```

Reply:
No need to look for file names with spaces. It is just a single file with a problematic file name. It is not in master any more. Removing it is better.